internal/rootcoord/meta_table.go

```diff
@@ -232,9 +232,9 @@ func (mt *metaTable) AddProxy(po *pb.ProxyMeta, ts typeutil.Timestamp) error {
 	defer mt.proxyLock.Unlock()
 
 	k := fmt.Sprintf("%s/%d", ProxyMetaPrefix, po.ID)
-	v := proto.MarshalTextString(po)
+	v, _ := proto.Marshal(po)
```

Comment:
better to handle this error

Reply:
because MarshalTextString does not return err, it's safe to replace with Marshal here